doc/release-process.md

```diff
@@ -241,15 +250,17 @@ cat "$VERSION"/*/all.SHA256SUMS.asc > SHA256SUMS.asc
        as save storage space *do not upload these to the bitcoincore.org server,
        nor put them in the torrent*.
 
-       ```sh
-       find guix-build-${VERSION}/output/ -maxdepth 2 -type f -not -name "SHA256SUMS.part" -and -not -name "*debug*" -exec scp {} user@bitcoincore.org:/var/www/bin/bitcoin-core-${VERSION} \;
-       ```
+       Wait until all of these files have finished uploading before uploading the SHA256SUMS(.asc) files.
 
     2. The `SHA256SUMS` file
 
-    3. The `SHA256SUMS.asc` combined signature file you just created
+    3. The `SHA256SUMS.asc` combined signature file you just created.
 
-- Create a torrent of the `/var/www/bin/bitcoin-core-${VERSION}` directory such
+- After uploading release candidate binaries, notify the bitcoin-core-dev mailing list and
+  bitcoin-dev group that a release candidate is available for testing. Include a link to the release
+  notes draft.
+
+- Create a torrent of the directory such
```

Comment:
If we're going to mention torrent generation, maybe also mention the OpenTimestamps file too? Both are generated automatically on the server now.

Reply:
Ah, I didn't even realize they were generated on the server. Edited.